scenariolib/visit.go

```diff
@@ -134,11 +134,16 @@ func (v *Visit) ExecuteScenario(scenario Scenario, c *Config) error {
 		if v.WaitBetweenActions {
 			var timeToWait int
 			if c.TimeBetweenActions > 0 {
-				timeToWait = c.TimeBetweenActions
+				timeToWait = c.TimeBetweenActions //Random waiting time
+				WaitBetweenActions(timeToWait)
+			}else if c.TimeBetweenActions < 0 {
+				timeToWait = c.TimeBetweenActions*(-1)
```

Comment:
You are right, that it would be better to use another parameter to control the randomness of the wait time. I propose to add a parameter "isWaitRandom" in the Config with a default value at true (the current behaviour). Then, simply add a new parameter to the WaitBetweenActions function: "waitBetweenActions(timeToWait, isWaitRandom)" and randomly wait or not. The code will become simpler to understand.